magento/magento2
# Fix the issue with missing asterisk for admin required fields

### Preconditions
There is a missing asterisk for admin required fields.

### Description (*)
Looks like the issue was accidentally caused by the following commit: https://github.com/magento/magento2/commit/7d4721dd38d832b072c2f77d74da3a23a53fa675

1. The same fix should be applied as for the Magento 2.3 in order to fix the issue: https://github.com/magento/magento2/commit/eb173dd1b90cf3c729ec10dd115652b56586108a
2. Also, the following temporary fix for admin login form should be reverted: https://github.com/magento/magento2/commit/af700494b83600a29d03badbef20374a81960b4a

### Fixed Issues
1. magento/magento2#18904: Missing asterisk for admin required fields

### Manual testing scenarios (*)
1. Check the admin form with required fields. (E.g. Admin login page, admin order create page etc.)

### Expected result
1. The asterisk is shown for the required foelds.
<img width="365" alt="screen shot 2018-10-28 at 12 26 13 pm" src="https://user-images.githubusercontent.com/11693779/47614897-d9064980-daaf-11e8-8a90-ed8486eb2d81.png">
<img width="822" alt="screen shot 2018-10-28 at 12 33 27 pm" src="https://user-images.githubusercontent.com/11693779/47614898-d9064980-daaf-11e8-8786-01ccf4c598de.png">

### Contribution checklist (*)
